### DISCUSSION TITLE
Support for a vanilla JS version

### DISCUSSION THREAD

**Discussion by mathuo** [mathuo/dockview#239] [Ideas]
As of version `1.7.0` the project has been split into two published pacakges `dockview` and `dockview-core`. `dockview-core` contains the underlying logic for the project whereas `dockview` contains the React code as well as depending on `dockview-core`.

> For those using `dockview` you do not need to worry about installing `dockview-core` too.

You can view an example written in Vanilla JS using `dockview-core` [here](https://dockview.dev/docs/components/dockview#vanillajs). The interfaces and classes in `dockview-core` are subject to change and may not always be intutative to use since this area is still a work in progress.

**Comment by BlueBazze** [comment]
A Vue 3 example, works with Nuxt 3 aswell.

https://gist.github.com/BlueBazze/19abc1b379486b45b4b57f7692372dd5
